libraries/ms-drivers/src/drv120_relay.c

```diff
@@ -1,10 +1,16 @@
 #include "drv120_relay.h"
+#include "log.h"
 
 // Storage for EN pin on DRV120 to open/close relay
 static GpioAddress s_relay_pin;
+static Drv120ErrorCallback s_callback;
 
-StatusCode drv120_relay_init(GpioAddress *pin) {
-  s_relay_pin = *pin;
+static void prv_drv120_gpio_it_cb(const GpioAddress *address, void *context) {
+  s_callback(context);
```

Comment:
Add a guard so we don't call `s_callback` if it's null to avoid segfaults. (User code *should* always set `error_handler` to something non-null if `status_pin` is set, but if they don't the resulting segfault here is near impossible to debug.)